packages/node_modules/@ciscospark/widget-recents/src/container.js

```diff
@@ -59,7 +43,7 @@ const injectedPropTypes = {
   incomingCall: PropTypes.object,
   media: PropTypes.object.isRequired,
   mercuryStatus: PropTypes.object.isRequired,
-  spaces: PropTypes.object.isRequired,
+  spacesById: PropTypes.object.isRequired,
```

Comment:
The selector was providing a `spaces` as well as the redux store being named `spaces` which was confusing and very error prone. Specifically defining what is coming from the selector is more intuitive.